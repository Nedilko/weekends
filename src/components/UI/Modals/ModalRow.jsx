import PropTypes from 'prop-types'

ModalRow.propTypes = {
  children: PropTypes.node.isRequired,
}

function ModalRow({ children }) {
  return (
    <div className="my-1 flex items-center justify-start gap-5">{children}</div>
  )
}

export default ModalRow
