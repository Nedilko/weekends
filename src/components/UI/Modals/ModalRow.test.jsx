import ModalRow from '@UI/Modals/ModalRow'
import { render } from '@utils/test-utils'

describe('modal row', () => {
  it('renders content', () => {
    const { container, getByText } = render(<ModalRow>some text</ModalRow>)
    expect(getByText('some text')).toBeInTheDocument()
    expect(container).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})
