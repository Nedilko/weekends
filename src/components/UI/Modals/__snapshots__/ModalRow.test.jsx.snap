// Vitest Snapshot v1

exports[`modal row > renders content 1`] = `
<div>
  <div
    class="my-1 flex items-center justify-start gap-5"
  >
    some text
  </div>
</div>
`;
